components/table/interface.ts

```diff
@@ -119,6 +119,7 @@ export interface ColumnType<RecordType> extends Omit<RcColumnType<RecordType>, '
   sortOrder?: SortOrder;
   defaultSortOrder?: SortOrder;
   sortDirections?: SortOrder[];
+  sortIcon?: React.ReactNode | ((sortOrder: SortOrder) => React.ReactNode);
```

Comment:
Should we drop `React.ReactNode` type? Is `((sortOrder: SortOrder) => React.ReactNode)` enough already?

Reply:
I agree with you and gonna erase it. Thank you so much.

Reply:
I've fixed it.
https://github.com/ant-design/ant-design/pull/42498/commits/bc4779aea8815c5e152d83704b0d7d4634b8cdbc